release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -447,6 +447,18 @@ We will have details here when {product-title} {product-version} is released.
 ==== Support for managing the Gateway API custom resource definition (CRD) lifecycle
 We will have details here when {product-title} {product-version} is released.
 
+[id="ocp-4-19-networking-ptp-dual-oc_{context}"]
+==== Dual-port NICs for improved redundancy in PTP ordinary clocks (Technology Preview)
+With this release, you can use a dual-port network interface controller (NIC) to improve redundancy for Precision Time Protocol (PTP) ordinary clocks.
+Available as a Technology Preview, in a dual-port NIC configuration for an ordinary clock, if one port fails, the standby port takes over, maintaining PTP timing synchronization.
```

Comment:
Again, not holding up the PR over it, but this is a lot of commas. I'd recommend breaking out the first piece about TP into its own sentence, like "This feature is available as a Technology Preview only"